tests/TestCase/Authenticator/CookieAuthenticatorTest.php

```diff
@@ -135,6 +125,34 @@ public function testCredentialsNotPresent()
         $this->assertEquals(Result::FAILURE_CREDENTIALS_NOT_FOUND, $result->getCode());
     }
 
+    /**
+     * testAuthenticateInvalidToken
+     *
+     * @return void
+     */
+    public function testAuthenticateInvalidToken()
+    {
+        $identifiers = new IdentifierCollection([
+            'Authentication.Password'
+        ]);
+
+        $request = ServerRequestFactory::fromGlobals(
+            ['REQUEST_URI' => '/testpath'],
+            null,
+            null,
+            [
+                'CookieAuth' => 'mariano:$2y$10$1bE1SgasKoz9WmEvUfuZLeYa6pQgxUIJ5LAoS/asdasdsadasd'
```

Comment:
Should we have a test for the valid password hash with a mismatched username on the left side of the token?